client/infinite_scroll.js

```diff
@@ -9,11 +9,16 @@ TemplateController('infinite_scroll', {
       instance.target.trigger("becameVisible");
     }
     };
+    this.showMoreVisibleDebounced = () => {
+     return (event) => {
+       _.debounce(this.showMoreVisible, this.data.timeDelay);
+     }
+   };
```

Comment:
You don't have to wrap it into a function, in fact this creates the same problem again!
Now you call this function twice, and it returns two separate debounced functions.

The key here is that you should call `_.debounce` only once, like this:
```javascript
this.showMoreVisibleDebounced = _.debounce(this.showMoreVisible, this.data.timeDelay);
```